documentation/book/proc-resizing-persistent-volumes.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-storage.adoc
+
+[id='proc-resizing-persistent-volumes-{context}']
+= Resizing persistent volumes
+
+You can provision increased storage capacity by increasing the size of the persistent volumes used by an existing {ProductName} cluster.
+Resizing persistent volumes is supported in clusters that use either a single persistent volume or multiple persistent volumes in a JBOD storage configuration.
+
+NOTE: You can increase but not decrease the size of persistent volumes. Decreasing the size of persistent volumes is not currently supported in {ProductPlatformName}.
+Decreasing the size of persistent volumes is currently not possible.
```

Comment:
You can remove line 12 because it's covered in the first sentence.

Reply:
Done, thanks.